CMakeLists.txt

```diff
@@ -55,14 +55,16 @@ target_include_directories(plutovg PUBLIC
 
 find_library(MATH_LIBRARY m)
 if(MATH_LIBRARY)
-    target_link_libraries(plutovg PRIVATE ${MATH_LIBRARY})
+    target_link_libraries(plutovg PRIVATE m)
```

Comment:
This changes not only how `m` is linked put also the export cmake config for static linkage. It is equivalent to what is exported with pkgconfig in the meson build.